bagua/torch_api/bucket.py

```diff
@@ -193,21 +225,54 @@ def append_decentralized_synchronous_op(
             hierarchical (bool): Enable hierarchical communication. Which means the GPUs on the same machine
                 will communicate will each other first. After that, machines do inter-node communication. This can
                 boost performance when the inter-node communication cost is high.
-            peer_selection_mode (str): Can be "all" or "shift_one". "all" means all workers'
-                weights are averaged in each communication step. "shift_one" means each worker
-                selects a different peer to do weights average in each communication step.
+            peer_selection_mode (str): Can be "all" or "shift_one" for full precision decentralized operation, while "ring" for
+                low precision decentralized operation. "all" means all workers' weights are averaged in each communication step.
+                "shift_one" means each worker selects a different peer to do weights average in each communication step.
+                "ring" means all workers are connected into a ring, and each worker communicate with its neighbors.
             communication_interval (int): Number of iterations between two communication steps.
+            compression: If not ``None``, the tensors will be compressed for communication. Currently "MinMaxUInt8" is
+                supported.
+            weight (torch.Tensor): Local model of current worker, required for low precision decentralized operation.
+            left_peer_weight (torch.Tensor): Model replica of current worker's connected left peer, required for low
```

Comment:
Need to explain what "model replica" is to make user understand how to use it.

A tensor with the same size as weight?

Reply:
@ganshaoduo any suggestions?